test/test_local_db.js

```diff
@@ -261,36 +264,8 @@ var testReplicaIncorrectPassword = function (cb) {
 }
 
 var testparseMongoConnectionString = function (cb) {
```

Comment:
This is now in the test `test errors should be thrown for invalid connection strings`